fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -809,6 +809,12 @@ public static void loadJournal(Catalog catalog, JournalEntity journal) {
                     catalog.getAnalyzeManager().replayRemoveAnalyzeJob(analyzeJob);
                     break;
                 }
+                case OperationType.OP_MODIFY_HIVE_TABLE_COLUMN:{
```

Comment:
If this log is only for hive table, please change the catalog.replayModifyTableColumn to catalog.replayModifyHiveTableColumn. If this is for all kind table, change OP_MODIFY_HIVE_TABLE_COLUMN to OP_MODIFY_TABLE_COLUMN